chrome/content/zotero/xpcom/utilities_internal.js

```diff
@@ -1700,6 +1700,52 @@ Zotero.Utilities.Internal = {
 		elem.appendChild(menu);
 		return menu;
 	},
+
+	// Show View > Columns, Sort By and Move Column menus for windows that have an itemTree
+	handleItemTreeMenuShowing: function (event, menuID, itemsView) {
+		let document = event.target.ownerDocument;
+		let menuPopup = document.getElementById(menuID).menupopup;
+		if (event.target !== menuPopup) {
```

Comment:
When I tried, `event.currentTarget` was `null` for me...

What if we pass the actual `menu` node instead of the `menuID`? Then, in `zoteroPane.xhtml`, we have `<menu id="sort-submenu" onpopupshowing="ItemTreeMenuBar.handleItemTreeMenuShowing(event, this, ZoteroPane.itemsView)">`.  And to check if it is the right popup  in `handleItemTreeMenuShowing`, we just do `event.target !== menu.menupopup`?